File: Universal Windows Platform Solution/Phantom of the West/Phantom of the West/Visual Novel/Chapters/Chapter 11/IWatchTV.cs
using Phantom_of_the_West.Visual_Novel.Serialization.Event_Flagging;

namespace Phantom_of_the_West.Visual_Novel.Chapters.Chapter_11
{
	internal class IWatchTV : StoryChoice
	{
		internal IWatchTV() : base(StoryChoice.GetLocalizedString("ch11IWatchTV"), 11132)
		{
		}

		public override void Select()
		{
			PotWVN vn = PotWVN.MainVN;
			vn.EventFlags.Ch11ActedInSarahRoom = ActivityInSarahRoom.WatchedTV;
			base.Select();
		}
	}
}

File: Android Project/app/src/main/java/com/christopherraleigh/phantomofthewest/visual_novel/chapters/chapter_11/IWatchTV.java
package com.christopherraleigh.phantomofthewest.visual_novel.chapters.chapter_11;

import com.christopherraleigh.phantomofthewest.R;
import com.christopherraleigh.phantomofthewest.visual_novel.PotWVN;
import com.christopherraleigh.phantomofthewest.visual_novel.chapters.StoryChoice;
import com.christopherraleigh.phantomofthewest.visual_novel.serialization.event_flagging.ActivityInSarahRoom;

/**
 * Created by christopherraleigh on 2016-05-22.
 */
public class IWatchTV extends StoryChoice {
    public IWatchTV() {
        super(R.string.ch11IWatchTV, 11132);
    }

    @Override
    public void select() {
        PotWVN vn = PotWVN.getMainVN();
        vn.getEventFlags().setCh11ActedInSarahRoom(ActivityInSarahRoom.WatchedTV);
        super.select();
    }
}
